global-test.properties

```diff
@@ -12,38 +12,7 @@ xtf.junit.prebuilder.synchronized=true
 # Bootable JAR OpenJDK base image
 intersmash.bootable.jar.image=registry.access.redhat.com/ubi8/openjdk-17
 
-# WildFly operator settings
-intersmash.wildfly.operators.catalog_source=community-operators-wildfly-operator
-intersmash.wildfly.operators.index_image=quay.io/operatorhubio/catalog:latest
-intersmash.wildfly.operators.package_manifest=wildfly
-intersmash.wildfly.operators.channel=alpha
-
-# Wildfly images
-intersmash.wildfly.image=quay.io/wildfly/wildfly-s2i-jdk17:latest
-intersmash.wildfly.runtime.image=quay.io/wildfly/wildfly-runtime-jdk17:latest
-
-# Wildfly Helm Charts
-intersmash.wildfly.helm.charts.repo=https://github.com/wildfly/wildfly-charts.git
-intersmash.wildfly.helm.charts.branch=wildfly-2.3.2
-intersmash.wildfly.helm.charts.name=wildfly
-
-# ActiveMQ operator settings
-# NOTE: we are using http://quay.io/jbossqe-eap for hosting our custom operator-catalog and operator-bundle images
-#       see https://github.com/Intersmash/intersmash/issues/32 for context
-intersmash.activemq.operators.catalog_source=intersmash-activemq-operator-index
-intersmash.activemq.operators.index_image=quay.io/jbossqe-eap/intersmash-activemq-operator-catalog:v1.0.11
-intersmash.activemq.operators.package_manifest=activemq-artemis-operator
-intersmash.activemq.operators.channel=upstream
-
-# Kafka operator settings
-intersmash.kafka.operators.channel=stable
-
```

Comment:
@fabiobrz for the sake of users easiness of getting started quickly, why don't we leave upstream defaults here or in `test.properties`? I was asking because, by default, we have `<intersmash.test.execution.profile>community</intersmash.test.execution.profile>`

Reply:
You're right, this makes sense. I'll fix it, thanks!